include/z64.h

```diff
@@ -1207,7 +1207,7 @@ typedef struct PlayState {
     /* 0x1241B */ u8 transitionMode; // "fbdemo_wipe_modem"
     /* 0x1241C */ TransitionFade transitionFade;
     /* 0x12428 */ char unk_12428[0x3];
-    /* 0x1242B */ u8 unk_1242B;
+    /* 0x1242B */ u8 viewpoint; // toggable camera states for houses and shops. Is also equal to the bgCamIndex + 1
```

Comment:
"toggable" ? ("toggleable" isn't a word either I don't think, may want to word this differently)

Also is "camera states" meant to be plural?

And maybe it's just because of the current wording or because the system can be used in a wide variety of ways, but this comment feels very vague, I guess this could be addressed by indicating what kind of "state" this refers to? (setting, mode, bgCamIndex, camera flag ...)

Reply:
I think `toggable` and `toggleable` both aren't words but are still commonly used informally so it's fine? `toggleable` is more common and I've been trying to stick with that, so I can change it if its not too out there for a more informal word.

An yeah `camera states` -> `camera setting`

Reply:
toggleable is for sure how id say it. and even if its not literally in the dictionary, its still a thing that people say.
 for a very small sample size (just our zeldaret discord) `toggleable` is used by a bunch of people in different contexts, and theres 1 instance of `toggable` from engineer last week which was probably a typo(?)

Reply:
yea toggable is almost never used (and fails chrome's spell check), whereas togglable and toggleable are far more common (and probably should be toggleable).

I don't think the comment completely expresses what the variable does. what does it mean for a viewpoint to be "toggleable"?

Reply:
c up to toggle viewpoint indoors (links house for example, overhead view or center/pivot view)

Reply:
Yeah, and I emphasize the `toggleable` part (my brain does actually want togglable), because this viewpoint variable isn't used at all in scenes with a locked fixed camera or scenes with a locked pivot camera (which you could think of as viewpoints). It's only used in scenes with the capability to toggle between that `fixed <-> pivot` cameras, i.e. in indoors (overhead view vs. center/pivot view, toggled through c-up) and also shops (side fixed view vs. shop browsing view, toggled through the shopkeeper (ossen)).